sway-core/src/language/call_path.rs

```diff
@@ -45,7 +45,11 @@ impl<T: Spanned> Spanned for CallPath<T> {
         if self.prefixes.is_empty() {
             self.suffix.span()
         } else {
-            let prefixes_spans = self.prefixes.iter().map(|x| x.span());
+            let prefixes_spans = self
+                .prefixes
+                .iter()
+                .map(|x| x.span())
+                .filter(|x| x.path() == self.suffix.span().path());
```

Comment:
Will you add a code comment above this to document that this LOC should be removed when #21 goes in?